packages/get-organisation/index.ts

```diff
@@ -0,0 +1,63 @@
+import { getOrganisationInfoAndStats } from "@create-disruptions-data/shared-ts/utils/dynamo";
```

Comment:
I think we could have both of these endpoints in the same package folder, maybe something like `organisations-api/get-organisation` and `organisations-api/get-organisations`. Then they can share the same package.json and config files etc.

Reply:
done